baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -481,6 +511,7 @@ export class BigQueryStorageClient {
     ] = gax.routingHeader.fromParams({
       'stream.name': request.stream!.name || '',
     });
+    this.initialize();
```

Comment:
this this potentially be:

```
this.initialize().then(() => {
  return this._innerApiCalls.splitReadStream(request, options, callback);
})
```

It seems like we're still potentially creating promises that are unhandled, or maybe I'm missing something?

Reply:
Yes, this does not change. We won't be able to make it perfect until we stop supporting callbacks. But now it can be more predictable since users can do `await client.initialize()` and catch all errors there.

Reply:
Another problem is that the method must immediately return a _cancellable call_ that comes from Gax, not just a promise that eventually resolves to a result. 

This use case:

```
const call = client.doStuff(request);
call.cancel();  // <---- existing feature, we don't want to break it now
...
call.then(result => ...);
```